content/python/concepts/math-module/terms/math-factorial/math-factorial.md

```diff
@@ -0,0 +1,53 @@
+---
+Title: 'math.factorial()'
+Description: 'Returns factorial of a number.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Arithmetic'
+  - 'Functions'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The `math.factorial()` method returns the factorial of a positive number.
```

Comment:
```suggestion
The `math.factorial()` method returns the factorial of a positive number.
```
The first mention of the concept must always be in **bold**. Please make this correction here.